addon/components/table-columns.js

```diff
@@ -32,6 +32,12 @@ export default Ember.Component.extend({
   */
   rowClasses: null,
 
+  /**
+    Name of data property for row groups
+    @public
+  */
+  rowGroupDataName: 'rowData',
```

Comment:
We should make this a property on `justa-table` and the property here should just be a `readOnly` alias to `table.rowGroupDataName` so the user specifies it on the top level table rather than each column set.